docs/sources/operations/storage/boltdb-shipper.md

```diff
@@ -48,22 +48,26 @@ Loki can be configured to run as just a single vertically scaled instance or as
 When it comes to reads and writes, Ingesters are the ones which writes the index and chunks to stores and Queriers are the ones which reads index and chunks from the store for serving requests.
 
 Before we get into more details, it is important to understand how Loki manages index in stores. Loki shards index as per configured period which defaults to 7 days i.e when it comes to table based stores like Bigtable/Cassandra/DynamoDB there would be separate table per week containing index for that week.
-In case of BoltDB files there is no concept of tables, so it creates a BoltDB file per period(i.e day in case of boltdb-shipper store). Files/Tables created per day are identified by a configured `prefix_` + `<period-number-since-epoch>`.
+In the case of BolDB Shipper, a table is defined by a collection of many smaller BoltDB files, each file storing just 15 mins worth of index. Tables created per day are identified by a configured `prefix_` + `<period-number-since-epoch>`.
 Here `<period-number-since-epoch>` in case of boltdb-shipper would be day number since epoch.
-For example, if you have prefix set to `loki_index_` and a write request comes in on 20th April 2020, it would be stored in table/file named `loki_index_18372` because it has been `18371` days since epoch, and we are in `18372`th day.
+For example, if you have a prefix set to `loki_index_` and a write request comes in on 20th April 2020, it would be stored in a table named loki_index_18372 because it has been `18371` days since the epoch, and we are in `18372`th day.
 Since sharding of index creates multiple files when using BoltDB, BoltDB Shipper would create a folder per day and add files for that day in that folder and names those files after ingesters which created them.
 
+To reduce the size of files which help with faster transfer speeds and reduced storage costs, they are stored after compressing them with gzip.
+
 To show how BoltDB files in shared object store would look like, let us consider 2 ingesters named `ingester-0` and `ingester-1` running in a Loki cluster, and
 they both having shipped files for day `18371` and `18372` with prefix `loki_index_`, here is how the files would look like:
 
 ```
 └── index
     ├── loki_index_18371
-    │   ├── ingester-0
-    │   └── ingester-1
+    │   ├── ingester-0-1587254400.gz
+    │   └── ingester-1-1587255300.gz
+    |   ...
     └── loki_index_18372
-        ├── ingester-0
-        └── ingester-1
+        ├── ingester-0-1587254400.gz
+        └── ingester-1-1587254400.gz
+        ...
 ```
 *NOTE: We also add a timestamp to names of the files to randomize the names to avoid overwriting files when running Ingesters with same name and not have a persistent storage. Timestamps not shown here for simplification*
```

Comment:
```suggestion
**Note:** We also add a timestamp to names of the files to randomize the names to avoid overwriting files when running Ingesters with same name and not have a persistent storage. Timestamps not shown here for simplification.
```